packages/editor/src/bindings/post-meta.js

```diff
@@ -7,22 +7,43 @@ import { store as coreDataStore } from '@wordpress/core-data';
  * Internal dependencies
  */
 import { store as editorStore } from '../store';
+import { unlock } from '../lock-unlock';
+
+function getMetadata( registry, context ) {
+	let metaFields = {};
+	const { type } = registry.select( editorStore ).getCurrentPost();
```

Comment:
There is also `getCurrentPostType`, which seems more appropriate here.

Reply:
Using `getCurrentPostType` looks better. In previous implementations, we were getting more properties from `getCurrentPost`, I guess that's why it is being used.

I can change that in a follow-up PR.

Reply:
I have created [this follow-up pull request](https://github.com/WordPress/gutenberg/pull/65449) to handle this. Let's discuss there any potential change.